script.sh

```diff
@@ -57,14 +59,34 @@ while true
                  EROP+="\n"$(sed -i 's/light-updates: false/light-updates: true/g' PocketMine-MP/src/pocketmine/resources/pocketmine.yml 2>&1)
                  psucc
                 ;;
-            d|D) echo -e "${RED}This function has not been added yet, come to the GitHub repo everyday and see when it gets finished!${NF}"
-                 EROP="Function not added yet"
-                 SUCC=no
+            d|D) echo -e "Building DevTools and PocketMine Phar"
+                 bin/php7/bin/php PocketMine-DevTools/src/DevTools/ConsoleScript.php --make PocketMine-DevTools --relative PocketMine-DevTools --out PocketMine-MP/plugins/DevTools.phar
+                 cd PocketMine-MP
+                 EROP=$(echo -e "version\nmakeserver\nstop\n" | ../bin/php7/bin/php src/pocketmine/PocketMine.php --no-wizard --disable-ansi --disable-readline --debug.level=2)
+                 cd ..
+                 if ls PocketMine-MP/plugins/DevTools/PocketMine*.phar >/dev/null 2>&1; then
+                     echo "Server phar created successfully."
+                     cp PocketMine-MP/plugins/DevTools/PocketMine*.phar pocketmine.phar
+                     echo "Your new phar is now called pocketmine.phar!"
+                 else
+                     echo "${RED}No phar created!"
+                 fi
+                 psucc
                 ;;
             e|E) echo "Deleting PocketMine-MP folder..."
                  rm -rf PocketMine-MP/
                  psucc
                 ;;
+            f|F) echo "Cloning PocketMine-DevTools from https://github.com/pmmp/PocketMine-DevTools.git..."
+                 EROP=$(git clone --recursive "https://github.com/pmmp/PocketMine-DevTools.git" 2>&1)
+                 psucc
+                ;;
+            g|G) echo "Download PHP7 from https://bintray.com/pocketmine/PocketMine/download_file?file_path=PHP_7.0.3_x86-64_Linux.tar.gz..."
```

Comment:
It's probably a better idea to download PHP from Azure DevOps or Jenkins because Bintray is outdated

Reply:
This was two years ago :|